src/main/java/com/sygic/travel/sdk/tours/facade/ToursFacade.kt

```diff
@@ -9,11 +10,18 @@ class ToursFacade internal constructor(
 	private val toursService: ToursService
 ) {
 	/**
-	 * Creates and sends a request to get the Tours.
-	 * @param toursQuery ToursQuery encapsulating data for API request.
+	 * Creates and sends a request to get the Viator Tours.
 	 */
-	fun getTours(toursQuery: ToursQuery): List<Tour>? {
+	fun getToursViator(toursViatorQuery: ToursViatorQuery): List<Tour>? {
 		checkNotRunningOnMainThread()
-		return toursService.getTours(toursQuery)
+		return toursService.getToursViator(toursViatorQuery)
+	}
+
+	/**
+	 * Creates and sends a request to get the Get Your Guide Tours.
+	 */
+	fun getToursGetYourGuide(toursGetYourGuideQuery: ToursGetYourGuideQuery): List<Tour>? {
```

Comment:
Make the return type non nullable now, please.

Reply:
✔️